datasets/sg/gov_dir/crawler.py

```diff
@@ -0,0 +1,158 @@
+                                .text_content()
+                                .strip()
+                            )
+                            email = (
+                                official.find(".//div[@class='email info-contact']")
+                                .text_content()
+                                .strip()
+                            )
+                            # phone numbers are also available
+                            is_ministry = (
+                                data_url == "https://www.sgdi.gov.sg/ministries"
+                            )
+                            is_board_member = (
+                                data_url == "https://www.sgdi.gov.sg/statutory-boards"
+                                and section_name.lower()
+                                in {"board members", "council members"}
+                            )
+                            if is_ministry:
+                                position = f"{position} in the {ministry}"
+                            elif is_board_member:
+                                position = f"{position} of the Board of the {agency}"
+                            else:
+                                position = f"{position} of the {agency}"
+
+                            position = re.sub(
+                                "Board Member of the Board of",
+                                "Member of the Board of",
+                                position,
+                                re.I,
+                            )
+                            print(f"Formatted Position: {position}")
+                            match = TITLE_REGEX.match(full_name)
+                            title = None
+                            if match:
+                                full_name = match.group("name")
+                                title = match.group(1)
+
+                            person = context.make("Person")
+                            person.id = context.make_id(full_name, position)
+                            person.add("name", full_name)
+                            person.add("sourceUrl", link)
+                            person.add("topics", "role.pep")
+                            if title is not None:
+                                person.add("title", title)
+                            if email is not None:
+                                if email.startswith("https://"):
+                                    person.add("website", email)
+                                else:
+                                    person.add("email", email)
+
+                            position = h.make_position(
+                                context,
+                                name=position,
+                                country="sg",
+                                # topics=["gov.executive", "gov.national"],
+                            )
+
+                            categorisation = categorise(context, position, is_pep=True)
```

Comment:
I think it would be good to have a function `is_pep(rank) -> bool | None`

any rank containing minister, president, member is true
any rank containing `pa to` etc is false
everything else is None, and we can use the categorisation UI to categorise them. e.g. `Manager (Events) in the PRIME MINISTER'S OFFICE`